universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -74,7 +74,7 @@ public String getApplicantName() {
       return firstName;
     } catch (NoSuchElementException e) {
       logger.error("Application {} does not include an applicant name.");
-      return "<Anonymous Applicant>";
+      return "Guest";
```

Comment:
suggestion (blocking): please use the messages API and add "Guest" as a string in the `us-EN` locale

Reply:
https://docs.civiform.us/contributor-guide/developer-guide/internationalization-i18n

Reply:
@azizova-leyla and I addressed this in the latest commits.